tests/conftest.py

```diff
@@ -146,13 +146,17 @@ def pytest_addoption(parser):
                      help="Run sigrok usb tests with fx2lafw device (0925:3881)")
     parser.addoption("--local-sshmanager", action="store_true",
                      help="Run SSHManager tests against localhost")
+    parser.addoption("--ssh-username", default=None,
+                     help="SSH username to use for SSHService testing")
```

Comment:
There is no `SSHService` (it should really be called that, but it's `NetworkService`). Below as well.